.github/workflows/aarambh_staging.yml

```diff
@@ -4,7 +4,12 @@ name: Aarambh Staging release
 
 # Controls when the action will run.
 on:
-  workflow_dispatch
```

Comment:
We still need the 'workflow_dispatch' trigger so we can deploy the helplines individually if required

Reply:
@stephenhand I am not sure the `workflow_dispatch` initially does anything. It was a stand-alone with nothing (like inputs) to trigger. It kept on breaking the code until I removed it.

Reply:
Without that we can't run the builds independently from the Github GUI (I tested this to confirm), so removing it is not an option. 

It's valid to have a `workflow_dispatch` without any options, you just have to pick a branch or tag to run it from, you need to fix whatever issues you were having that caused you to remove it initially

Reply:
@stephenhand Fixed!